pubspec.yaml

```diff
@@ -80,6 +80,7 @@ dependencies:
 
   # functional programming thingies
   dartz: 0.10.1
+  upgrader: ^4.11.1
```

Comment:
this should probably have it own category and not fall under 'functional programming thingies'